src/pages/products/products.jsx

```diff
@@ -1,12 +1,150 @@
-import react from "react";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
 import { Footer } from "../../components/footer/footer";
```

Comment:
try to use absolute imports. Looks clean and will be easy to manage if its a big project.